pallets/gear/src/manager.rs

```diff
@@ -32,17 +32,23 @@ use frame_support::{
 use gear_core::{
     memory::PageNumber,
     message::{Dispatch, ExitCode, MessageId},
-    program::{Program as NativeProgram, ProgramId},
+    program::{CodeHash, Program as NativeProgram, ProgramId},
 };
 use primitive_types::H256;
 use sp_runtime::{
     traits::{UniqueSaturatedInto, Zero},
     SaturatedConversion,
 };
-use sp_std::{collections::btree_map::BTreeMap, marker::PhantomData, prelude::*};
+use sp_std::{
+    collections::{btree_map::BTreeMap, btree_set::BTreeSet},
+    marker::PhantomData,
+    prelude::*,
+};
 
 pub struct ExtManager<T: Config> {
     _phantom: PhantomData<T>,
+    // Messages with these destinations will be forcibly pushed to the queue.
+    marked_destinations: BTreeSet<ProgramId>,
```

Comment:
I prefer trailing phantoms
```suggestion
    // Messages with these destinations will be forcibly pushed to the queue.
    marked_destinations: BTreeSet<ProgramId>,
    _phantom: PhantomData<T>,
```

Reply:
done